com.unity.netcode.gameobjects/Components/NetworkAnimator.cs

```diff
@@ -156,7 +156,7 @@ internal NetworkAnimatorStateChangeHandler(NetworkAnimator networkAnimator)
     /// <summary>
     /// NetworkAnimator enables remote synchronization of <see cref="UnityEngine.Animator"/> state for on network objects.
     /// </summary>
-    [AddComponentMenu("Netcode/" + nameof(NetworkAnimator))]
+    [AddComponentMenu("Netcode/Network Animator")]
```

Comment:
Just checking, are we OK without an ordering value? I guess, since it was before, but double-checking.

Reply:
I left orders alone but honestly, they probably don't need to be there. The whole list is showing up in alphabetical order which seems good to me 🤷